static/editor.js

```diff
@@ -107,6 +108,13 @@ define(function (require) {
                 this.eventHub.emit('modifySettings', {
                     compileOnChange: !this.settings.compileOnChange
                 });
+                new Alert().notify('Compile on code change has been toggled ' + (this.settings.compileOnChange ? 'ON' : 'OFF'), {
+                    "group": "togglecompile",
```

Comment:
In Javascript (unlike Python!) one doesn't need to put the key in "quotes" unless it isn't a single token. I'm surprised the linter didn't complain :)